k8s/voucher/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "${APP_INSTANCE_NAME}-tester"
+  labels:
+    app.kubernetes.io/name: "${APP_INSTANCE_NAME}"
+  annotations:
+    marketplace.cloud.google.com/verification: test
+spec:
+  serviceAccountName: ${SERVICE_ACCOUNT}
+  containers:
+  - name: tester
+    image: "${testerImage}"
+    imagePullPolicy: Always
+    env:
+    - name: NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    - name: SERVICEACCOUNT
+      valueFrom:
+        configMapKeyRef:
+          name: ${APP_INSTANCE_NAME}-deployer-config
+          key: SERVICE_ACCOUNT
+  restartPolicy: Never
```

Comment:
It's missing a newline at end of all the files.

Reply:
done.